_posts/2020/2020-04-23-instagram-blog-post.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: 5 tips to keep your company Instagram channel alive / 5 tips on how to manage your company Instagram channel
+date: 2020-06-04
+header_image: public/instagram-company-account.jpg
+header_position: center
+header_overlay: true
+category: techcomm-and-socialmedia
+tags: ["social media", "Instagram", "content"]
+authors: ["Aroso"]
+about_authors: ["amahaiuddin"]
+---
+
+It is called a picture is worth a thousand words - Instagram is the perfect social media platform for sharing visuals around the world.
+We started our Instagram channel[@life_at_epages](https://www.instagram.com/life_at_epages/){:target="_blank"} in 2019 to present our company culture and express thus who we are.
+But managing a company Instagram channel and coordinating content involves some important aspects to consider.
+In the following I will explain which actions we take in order to keep our company Instagram channel alive.
+
+## Create Instagram How-to guide
+
+Usually, an Instagram company channel is not run by a single person but by a team consisting of two or more employees.
+In order to develop and create consistent content for our Instagram channel, we created an Instagram How-to guide.
```

Comment:
Would make it more "guide" again. E.g. "In order to develop and create consistent content for the channel, it can be helpful to come up with an Instagram how-to guide that is agreed and shared with all involved employees. "